src/web/app/components/question-types/question-edit-details-form/option-rich-text-editor/option-rich-text-editor.component.html

```diff
@@ -1,4 +1,4 @@
 <editor class="editor" [ngClass]="{ 'editor-disabled': isDisabled }" [ngStyle]="{ 'min-height.px': minHeightInPx }"
-        [init]="init" [disabled]="isDisabled" [inline]="isInlineMode"
+        [init]="init" [disabled]="isDisabled" [inline]="isInlineMode" [ngModelOptions]="{ updateOn: 'blur' }"
```

Comment:
Might be a potential bug here. If it is only updated on 'blur':
- User enters something in the edit box but don't click outside.
- Click save immediately
- The content might not be recorded and updated.

Reply:
I tested that scenario quite a few times. Didn't see any bug there. I think it's because the mouse click down on the button would call the `onblur`, and then the mouse click up would call the save function.

This might be an issue with stuffs like selenium though. But I don't think it will affect users.